ygen/protogen.go

```diff
@@ -337,17 +463,18 @@ func writeProto3Msg(msg *yangDirectory, msgs map[string]*yangDirectory, state *g
 // package for the protobuf message(s) that are being generated, such that relative
 // paths can be used in the messages.
 // TODO(robjs): Split the logic of this function into multiple subfunctions.
-func genProto3Msg(msg *yangDirectory, msgs map[string]*yangDirectory, state *genState, cfg protoMsgConfig, parentPkg string) ([]protoMsg, []error) {
+func genProto3Msg(msg *yangDirectory, msgs map[string]*yangDirectory, state *genState, cfg protoMsgConfig, parentPkg string, childMsgs []*generatedProto3Message) ([]protoMsg, []error) {
```

Comment:
Consider []error -> Errors, here and elsewhere.

Reply:
Done.